recipes/default.rb

```diff
@@ -1,6 +1,7 @@
-ruby_block "Save node attributes" do
```

Comment:
How about using a directory and file resource instead of a ruby_block? Looking back on this, I wish I had done that.